changelogs/unreleased/47145-fix-filter-partner.yml

```diff
@@ -0,0 +1,3 @@
+...
+title :"fix filter in partner"
```

Comment:
Please add a bit more detail and add the main concerned object
Like `MOVE : Fix partner not being filtered according to payment mode compatible types`